gems/aws-sdk-core/spec/api_helper.rb

```diff
@@ -13,6 +13,14 @@ def sample_json # dynamodb
       end
     end
 
+    def sample_json_aws_query_compatible # sqs
```

Comment:
I think we should remove this - it won't get much re-use. I would prefer if, in error_handler_spec, we define a minimal API. There are some examples (endpoint_discovery_spec or all in that plugins folder).

Reply:
Thanks a lot for advice! Updated.